trigger.js

```diff
@@ -105,10 +108,30 @@ class TriggerHappy {
         this._executeTriggers(triggers);
     }
 
-    _onUpdateJournal(journal) {
-        if (journal.id === this._journalId || journal.name === this.journalName)
+    _onUpdateJournal(journal, update) {
+        if (update._id === this._journalId || journal.name === this.journalName)
             this._parseJournal();
     }
+
+    _onEndUpdate(scene, userId, update) {
+        if (update["x"] === undefined && update["y"] === undefined) return true;
+        let sceneTokens = scene.data.tokens;
+        let token = sceneTokens.find(t=>t._id === update._id);

```

Comment:
Use spaces to separate the arguments and arrow function body `(t => t._id === update._id)`